unit-test-runner/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16.17.1-buster
+
+# Install necessary dependencies
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
+    && apt-get update \
+    && apt-get install -y google-chrome-stable \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /palladium-view
+COPY . /palladium-view
+WORKDIR /palladium-view
+
+RUN npm install
+
+ENTRYPOINT ["npm"]
+CMD ["run", "ng", "test"]
```

Comment:
Question - do we really need a separate dockerfile to run unit tests?
Can we do it without docker - just install all deps inside GitHub Actions and run some comments?

I think that way we will be little bit more flexible in future - if we need we can create a dockerfile later
But having this command as native commands will allow us to run unit-tests easily on dev machine while development some feature without a need to constant rebuilding the dockerfile to rerun all the tests

In my option we need Dockerfile only for some production check, but unit-tests can be done on machined/GitHub Action directly